pom.xml

```diff
@@ -237,6 +237,10 @@
                 <enabled>false</enabled>
             </snapshots>
         </repository>
+        <repository>
```

Comment:
This needs to be removed when you're done with staging. I assume this is the case and that I should properly release 1.0.0.Beta2-SP1, which I will do now

Reply:
Great.  I will amend once that is complete.